Content.Tests/DMProject/Tests/Builtins/splicetext_runtime.dm

```diff
@@ -0,0 +1,21 @@
+/proc/RunTest()
+	try
+		splicetext("banana", 12, -1, "test") //bad text or out of bounds
+		ASSERT("THIS TEST SHOULD NOT PASS" == 0)
```

Comment:
`ASSERT()` just throws an exception when it fails, it would be caught by the try/catch. Maybe a framework like this instead:

```
var/passed = TRUE

try
    splicetext(...)
    passed = FALSE
catch

try
    splicetext(...)
    passed = FALSE
catch

ASSERT(passed)
```